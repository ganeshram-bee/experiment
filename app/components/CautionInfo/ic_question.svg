<svg xmlns="http://www.w3.org/2000/svg" width="12.018" height="12.018" viewBox="0 0 12.018 12.018">
    <defs>
        <style>
            .cls-1{fill:#f2b100}.cls-2{fill:#6d6daa}
        </style>
    </defs>
    <g id="ic_question" transform="translate(-58 -498)">
        <path id="ic_question-2" d="M-7.014-.041a5.84 5.84 0 0 1-2.338-.472A6.007 6.007 0 0 1-11.259-1.8a6.246 6.246 0 0 1-1.292-1.914 5.611 5.611 0 0 1-.465-2.338 5.807 5.807 0 0 1 .472-2.324 6.147 6.147 0 0 1 1.285-1.914 6.007 6.007 0 0 1 1.907-1.292 5.84 5.84 0 0 1 2.338-.472 5.855 5.855 0 0 1 2.331.472A6.043 6.043 0 0 1-2.769-10.3a6.16 6.16 0 0 1 1.292 1.905A5.768 5.768 0 0 1-1-6.057a6.01 6.01 0 0 1-.485 2.338A6.147 6.147 0 0 1-2.769-1.8 6.007 6.007 0 0 1-4.676-.513a5.84 5.84 0 0 1-2.338.472zm5.332-6a4.979 4.979 0 0 0-.41-2.078 5.525 5.525 0 0 0-1.142-1.7 5.381 5.381 0 0 0-1.688-1.148 5.1 5.1 0 0 0-2.064-.424 5.258 5.258 0 0 0-2.085.417 5.406 5.406 0 0 0-1.7 1.135 5.36 5.36 0 0 0-1.148 1.7 5.185 5.185 0 0 0-.424 2.085 5.013 5.013 0 0 0 .4 2.064 5.451 5.451 0 0 0 1.143 1.7 5.429 5.429 0 0 0 1.688 1.155 5.055 5.055 0 0 0 2.058.424 5.307 5.307 0 0 0 2.092-.417A5.328 5.328 0 0 0-3.254-2.27a5.465 5.465 0 0 0 1.148-1.7 5.136 5.136 0 0 0 .424-2.073z" class="cls-1" data-name="ic_question" transform="translate(71.016 510.059)"/>
        <path id="ic_question-3" d="M-8.859-7.342q-.164-.342-.342-.677t-.355-.69a.112.112 0 0 0 .034-.007.086.086 0 0 0 .034-.021 5.888 5.888 0 0 1 1.668-.608 4.506 4.506 0 0 1 1.764.007 2.5 2.5 0 0 1 .923.39 1.486 1.486 0 0 1 .567.827 2.046 2.046 0 0 1 .048 1 1.871 1.871 0 0 1-.485.9 2.144 2.144 0 0 1-.3.273q-.164.123-.328.26-.109.082-.232.164l-.246.164a.991.991 0 0 0-.308.308.6.6 0 0 0-.075.431.151.151 0 0 1-.007.048.207.207 0 0 0-.007.062h-1.519a2.9 2.9 0 0 1 .041-.937 1.308 1.308 0 0 1 .533-.772l.3-.2q.15-.1.3-.212a1.9 1.9 0 0 0 .185-.15 2.112 2.112 0 0 0 .171-.178.711.711 0 0 0 .144-.287.51.51 0 0 0-.034-.314.406.406 0 0 0-.205-.212 1.218 1.218 0 0 0-.3-.089 2.206 2.206 0 0 0-.574.021 2.993 2.993 0 0 0-.574.157q-.205.068-.4.157zm1.668 3.377a.769.769 0 0 1 .574.239.8.8 0 0 1 .232.581.785.785 0 0 1-.239.588.787.787 0 0 1-.567.232.81.81 0 0 1-.574-.232.774.774 0 0 1-.246-.588.8.8 0 0 1 .232-.581.785.785 0 0 1 .588-.239z" class="cls-2" data-name="ic_question" transform="translate(70.998 510.059)"/>
    </g>
</svg>
