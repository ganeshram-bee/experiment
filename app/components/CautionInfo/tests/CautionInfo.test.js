import React from 'react';
import { render } from 'react-testing-library';
import { generateImage } from 'jsdom-screenshot';

import CautionInfo from '../CautionInfo';

describe('<CautionInfo />', () => {
  
  // it('has no visual regressions', async () => {
  //   render(<CautionInfo />);

  //   const screenshot = await generateImage();
  //   expect(screenshot).toMatchImageSnapshot();
  // });

  it('should match the snapshot', () => {
    const { container } = render(<CautionInfo />);
    expect(container.firstChild).toMatchSnapshot();
  });

  // it('should match the snapshot', () => {
  //   const renderedComponent = renderer.create(<CautionInfo />).toJSON();
  //   expect(renderedComponent).toMatchSnapshot();
  // });

});
