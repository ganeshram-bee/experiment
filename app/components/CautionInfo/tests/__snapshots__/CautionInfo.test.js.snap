// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CautionInfo /> should match the snapshot 1`] = `
<div
  class="CautionInfo__LabelCon-sc-1x72tc4-0 eiwuiz"
  data-reactroot=""
>
  <table>
    <tbody>
      <tr>
        <td>
          <img
            alt="Info image"
            src="IMAGE_MOCK"
          />
        </td>
        <td>
          <p
            class="CautionInfo__InfoP-sc-1x72tc4-1 hKrsyG"
          >
            <span>
              Important Instructions
            </span>
          </p>
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
